src/Goerli-DssSpell.sol

```diff
@@ -18,11 +18,14 @@ pragma solidity 0.6.12;
 
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
+import "lib/dss-interfaces/src/dapp/DSTokenAbstract.sol";
+import "lib/dss-interfaces/src/dss/VatAbstract.sol";
```

Comment:
nit: you can use multiple imports:
`import { DSTokenAbstract, VatAbstract } from "dss-interfaces/Interfaces.sol";`

Reply:
Good point.